synapse/storage/data_stores/main/schema/full_schemas/README.md

```diff
@@ -1,13 +1,23 @@
 # Building full schema dumps
 
-These schemas need to be made from a database that has had all background updates run.
+These schemas need to be made from a database that has had all background
+updates run. The schemas are used at ...
 
 To do so, use `scripts-dev/make_full_schema.sh`. This will produce
-`full.sql.postgres ` and `full.sql.sqlite` files.
+`full.sql.postgres ` and `full.sql.sqlite` files. It generates SQL files for
+creating a brand new Synapse DB with the latest schema, on both SQLite3 and
+Postgres. It does so by having Synapse generate an up-to-date SQLite DB, then
```

Comment:
do we need this here as well as in the comments in the script itself? Having it in two places feels like a recipe for it getting out of date.

Reply:
I think, this readme is the right place for such information, maybe we can delete the info in the script top, but this especially doesent seem like some text that gets outdated

Reply:
I'm afraid I disagree. How it works is irrelevant here.